### PR TITLE
Mattb/swcd 2429 asset download component

### PR DESCRIPTION
**Pull request by mattb-hippo**
It's the asset download component:

- rendering code and contentful plumbing
- necessary stylesheets
- Contentful schema updates
- unit tests

### CODE REVIEW

File: Childrens-Social-Care-CPD/Views/Shared/_AssetDownload.cshtml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        {
+        <p class="gem-c-attachment__metadata">
+            This file is in an <a href="https://www.gov.uk/guidance/using-open-document-formats-odf-in-your-organisation" target="_blank" rel="noopener" class="govuk-link">OpenDocument</a> format
+        </p>    
+        }
+
+        <p class="gem-c-attachment__metadata">This file may not be suitable for users of assistive technology.</p>
+        <details data-module="ga4-event-tracker" data-ga4-event="{&quot;event_name&quot;:&quot;select_content&quot;,&quot;type&quot;:&quot;detail&quot;,&quot;text&quot;:&quot;Request an accessible format.&quot;,&quot;section&quot;:&quot;Request an accessible format.&quot;,&quot;index_section&quot;:1,&quot;index_section_count&quot;:1}" class="gem-c-details govuk-details govuk-!-margin-bottom-3" data-ga4-event-tracker-module-started="true">
+            <summary class="govuk-details__summary">
+                <span class="govuk-details__summary-text" data-ga4-expandable="">Request an accessible format.</span>
```
Comment (Maria-C1) [inline Childrens-Social-Care-CPD/Views/Shared/_AssetDownload.cshtml:148]: Remove `.` from the string in line 148 so it reflects designs.

**Review by Maria-C1 — APPROVED** [review]

File: Childrens-Social-Care-CPD/Views/Shared/_AssetDownload.cshtml (hunk 1 above)
Comment (mattb-hippo) [reply Childrens-Social-Care-CPD/Views/Shared/_AssetDownload.cshtml]: Done :)

**Pull request merged**